src/dc_context.c

```diff
@@ -376,6 +376,7 @@ static char* get_sys_config_str(const char* key)
  * - `e2ee_enabled` = 0=no end-to-end-encryption, 1=prefer end-to-end-encryption (default)
  * - `mdns_enabled` = 0=do not send or request read receipts,
  *                    1=send and request read receipts
+ * - `save_mime_headers` = set this to "1" if you want to use dc_get_mime_headers() later
```

Comment:
wasn't there a list somewhere returned by ``sys.config_keys``?

Reply:
f*ck ...

Reply:
you warned against it ;)